packages/babel-standalone/README.md

```diff
@@ -16,11 +16,11 @@ It's true that using Babel through Webpack, Browserify or Gulp should be suffici
 Installation
 ============
 
-There are several ways to get a copy of babel-standalone. Pick whichever one you like:
+There are several ways to get a copy of @babel/standalone. Pick whichever one you like:
```

Comment:
I think some of this text should still say `babel-standalone`, what do you think?

Reply:
maybe just "standalone Babel"? 🤔